x-pack/legacy/plugins/siem/public/components/event_details/columns.tsx

```diff
@@ -183,9 +183,10 @@ export const getColumns = ({
   },
   {
     field: 'valuesConcatenated',
+    name: i18n.BLANK,
```

Comment:
@andrew-goldstein does this look ok to you? i think this is a hidden (`1px`) column, so I just put a blank `name` as it is a required prop

Reply:
Yes this looks good. The contents of this field are not rendered. It exists to allow filtering on multiple values like IP addresses, when those values are encoded as an array instead of a string. I ran this branch locally and verified that filtering continues to work for hosts that have multiple IP addresses, i.e. `siem-es`.